src/include/cursor.h

```diff
@@ -109,8 +109,8 @@ struct __wt_cursor_btree {
     WT_REF *ref;   /* Current page */
     uint32_t slot; /* WT_COL/WT_ROW 0-based slot */
 
-    WT_INSERT_HEAD *ins_head; /* Insert chain head */
-    WT_INSERT *ins;           /* Current insert node */
+    wt_shared WT_INSERT_HEAD *ins_head; /* Insert chain head */
```

Comment:
I don't think this is shared. This is a thread local information for the cursor.

Reply:
Yep, good point. These were added due to [`ORDERED_READ`s in column_inline](https://github.com/wiredtiger/wiredtiger/blob/1befa2b41caad250425a84509c2464dd908f0ab6/src/include/column_inline.h#L24), but the `SKIP_FIRST`/`SKIP_LAST` macros mean the contents of ins_head should be annotated instead.
Fixed in 1a67820